eisbuk-admin/src/components/atoms/AttendanceSheet/AttendanceSheet.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import TableContainer from "@material-ui/core/TableContainer";
+import Paper from "@material-ui/core/Paper";
+import { CustomerWithAttendance, SlotInterface } from "@/types/temp";
+import AttendanceSheetSlot from "./AttendanceSheetSlot";
+
+interface SlotWithCustomers extends SlotInterface {
+  /**
+   * Record of customers who have booked (or are manually as attended),
+   * keyed by `customerId` and having all of the data for the customer plus
+   * values for `bookedInterval` and `attendedInterval`.
+   */
+  customers: CustomerWithAttendance[];
+}
+export interface Props {
+  date: SlotInterface["date"];
```

Comment:
The date, in this case, will be passed from redux store's `getCalendarDay`, so `DateTime`